src/lib/wordpress-provider/playground-cli/playground-cli-provider.ts

```diff
@@ -161,37 +174,47 @@ export class PlaygroundCliProvider implements WordPressProvider {
 
 				try {
 					await recursiveCopyDirectory( bundledWPPath, path );
-					serverOptions.wpVersion = this.DEFAULT_WORDPRESS_VERSION;
-					serverOptions.siteLanguage = DEFAULT_LOCALE;
-					serverOptions.isSetupMode = false;
-					serverOptions.isWpAutoUpdating = true;
-					serverOptions.blueprint = undefined;
 				} catch ( error ) {
 					throw new Error(
 						`Failed to copy WordPress files for offline setup: ${ ( error as Error ).message }`
 					);
 				}
-			}
 
-			await keepSqliteIntegrationUpdated( path );
+				setupSteps.push( {
+					step: 'runPHP',
+					code: `<?php
+					$_POST = array(
+						'language' => '${ this.escapePhpString( DEFAULT_LOCALE ) }',
+						'prefix' => 'wp_',
+						'weblog_title' => '${ this.escapePhpString( name ) }',
+						'user_name' => 'admin',
+						'admin_password' => '${ this.escapePhpString( adminPassword || '' ) }',
+						'admin_password2' => '${ this.escapePhpString( adminPassword || '' ) }',
+						'Submit' => 'Install WordPress',
+						'pw_weak' => '1',
+						'admin_email' => 'admin@localhost.com',
+					);
+					$_REQUEST = $_POST;
+					$_GET['step'] = 2;
 
-			const serverInstance = await this.startServer( serverOptions );
-			serverProcess = this.createServerProcess( serverInstance );
-			await serverProcess.start();
+					// Include WordPress installation
+					require_once('/wordpress/wp-admin/install.php');
+				`,
+				} );
+			}
 
-			if ( ! serverOptions.isSetupMode ) {
-				await this.runWordPressInstallation( serverProcess, serverOptions );
+			if ( ! server.meta.blueprint ) {
+				server.meta.blueprint = {};
 			}
+			const existingSteps = ( server.meta.blueprint.steps as unknown[] ) || [];
```

Comment:
Let's remove the `unkown` force type.

```suggestion
			const existingSteps = server.meta.blueprint.steps || [];
```

You can use this type definition in site-server to avoid using the unkown:

```diff
diff --git a/src/site-server.ts b/src/site-server.ts
index fbaa8cf4..8afe159f 100644
--- a/src/site-server.ts
+++ b/src/site-server.ts
@@ -1,6 +1,7 @@
 import fs from 'fs';
 import nodePath from 'path';
 import * as Sentry from '@sentry/electron/main';
+import { BlueprintV1Declaration } from '@wp-playground/blueprints';
 import fsExtra from 'fs-extra';
 import { parse } from 'shell-quote';
 import { portFinder } from 'common/lib/port-finder';
@@ -51,7 +52,7 @@ function getAbsoluteUrl( details: SiteDetails ): string {
 // We use SiteDetails for storing it in appdata-v1.json, so this meta was introduced for extra data which is not stored locally
 type SiteServerMeta = {
 	wpVersion?: string;
-	blueprint?: Record< string, unknown >;
+	blueprint?: BlueprintV1Declaration;
 };
 
 export class SiteServer {
```